src/pages/studyView/chartHeader/styles.scss

```diff
@@ -0,0 +1,69 @@
+.controls {
+    display: flex;
+}
+
+.fa-undo {
+    color: #2986e2;
+}
+
+.study-view-chart-contorls {
+    color: grey !important;
+    font-size: 12px;
+    float: right;
+    >button {
+        border: 0px !important;
+        cursor: pointer !important;
+        background-color: white !important;
+        background-image:none !important;
+        padding: 0px !important;
+        margin: 0 2px !important;
+        &:hover, &:active, &:focus {
+            color: grey !important;
+        }
+    }
+    >button:first-child {
+        margin-left: 10px !important;
+    }
+    .fa-arrows:hover {
+        cursor: move;
+    }
+}
+
+main {
```

Comment:
remember that these are global, meaning e.g. "main" will apply outside of studyview.     just because it's inside a file specific to study view doesn't scope it.  you should scope this entire file by wrapping it in an outer class .study-view or some such.

Reply:
also, main is not descriptive enough.